src/app/scripts/cac/pages/cac-pages-home.js

```diff
@@ -158,7 +205,7 @@ CAC.Pages.Home = (function ($, BikeModeOptions,  MapControl, Templates, UserPref
         event.preventDefault();
 
         // hide existing destinations list and show loading spinner
-        $(options.selectors.destinationsContainer).addClass('hidden');
+        $(options.selectors.placeList).addClass('hidden');
```

Comment:
There's no "View all destinations" link on the redesigned home page.  Is there a reason to keep this function (and the related selector and listener) around?